docs/portal.properties-Reference.md

```diff
@@ -50,7 +50,46 @@ skin.show_news_tab=
 skin.show_tools_tab=
 skin.show_about_tab=
 ```
-*Note: `skin.show_tools_tab` refers to the `Visualize Your Data` tab, while `skin.show_data_tab` refers to the `Data Sets` tab.
+
+**Note:** `skin.show_tools_tab` refers to the `Visualize Your Data` tab, while `skin.show_data_tab` refers to the `Data Sets` tab.
+
+## Quick Search (BETA)
+![Quick search example](images/previews/quick_search_example.png)
+
+All settings related to quick search (currently in beta):
+
+```
+# Enable/Disable quick search (default is false)
+quick_search.show=true
+# for studies queried when typing in a single gene use this when session
+# service is disabled: 
+quick_search.gene_query.cancer_study_list=
+quick_search.gene_query.cancer_study_list_name=
+# or this if you do have session service (title will be pulled from the
+# virtual_study):
+quick_search.gene_query.session_id=
+```
+
+One can show quick search by setting `quick_search.show=true` (default is
+false). The studies being queried when querying a gene is configured by
+specifying a cancer study list e.g.
+
+```
+quick_search.gene_query.cancer_study_list=laml_tcga_pan_can_atlas_2018,uvm_tcga_pan_can_atlas_2018
+quick_search.gene_query.cancer_study_list_name="TCGA PanCan Atlas studies"
+```
+
+or by specifying a session id of a virtual study:
+
+```
+quick_search.gene_query.session_id=5c8a7d55e4b046111fee2296
+```
+
+When using a session id, the title will be pulled from the virtual study. Make
+sure to create a `virtual_study` with studies that everybody has access to and
+don't use a `main_session` id.
+
+**Note:** When `quick_search.gene_query.cancer_study_list` or `quick_search.gene_query.session_id` is defined it will also be used as the default study for the `/ln?=TP53:MUT` links.
```

Comment:
do you think it's weird that we are embedding a default that can be used outside of quick search inside quicksearch config?  maybe we should have a default_cross_cancer_study_list that is used by ln and quick search?

Reply:
@alisman yeah was thinking that as well. We'll prolly need a `default_cross_cancer_session_id` config as well then